src/assets/stylesheets/components/_alert.scss

```diff
@@ -1,7 +1,6 @@
 .alert {
   display: flex;
   align-items: flex-start;
-  width: 100%;
   margin-bottom: rem(20px);
   padding: rem(20px);
```

Comment:
Sorry that I missed this before. But here the padding should be `16px` and margin-bottom should be `24px`
![image](https://user-images.githubusercontent.com/14927672/123210959-99a08d00-d4e4-11eb-9026-28b81125c032.png)

Reply:
Fixed in f750a80ec15769171eb3e14e9c74cf685010f945 and dfc347a19e0d3708f8f3bc409aa7b4c336d9d2fc